modules/snapshot/README.md

```diff
@@ -13,15 +13,15 @@ Terraform 0.12 and newer. Submit pull-requests to `main` branch.
 
 ```hcl
 module "observe_lambda" {
-  source           = "../.."
+  source           = "observeinc/lambda/aws"
   observe_customer = var.observe_customer
   observe_token    = var.observe_token
   observe_domain   = var.observe_domain
   name             = random_pet.run.id
 }
 
 module "observe_lambda_snapshot" {
-  source = "../../snapshot"
+  source = "observeinc/lambda/aws//modules/snapshot"
```

Comment:
Is there a reason some of the source strings start with 'observeinc/lambda/aws'? it looks like a local filesystem source to me, but then this would error

Reply:
`observeinc/lambda/aws` is a module registry reference. In the absence of a hostname, it defaults to `registry.terraform.io`. 

We host this module in the official terraform registry (https://registry.terraform.io/modules/observeinc/lambda/aws). I'm just making sure that the example READMEs reference that rather than a local path, so users can copy paste something more usable.